src/index.html

```diff
@@ -1,13 +1,59 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+
+      <div class="card__picture">
+        <img
+          src="./images/imac.jpeg"
+          alt="apple_blok"
+          class="card__picture--image"
+        >
+      </div>
+
+      <div class="card__goods-name">
+        <p class="card__goods-name--text">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </p>
+      </div>
+      <div class="card__goods-id">
+        <p class="card__goods-id--text">Код товара: 195434</p>
+      </div>
+
+      <div class="card__stars">
+        <div class="card__stars--4">
+          <div class="stars--4__item"></div>
+          <div class="stars--4__item"></div>
+          <div class="stars--4__item"></div>
+          <div class="stars--4__item"></div>
+          <div class="stars--4__item"></div>
+        </div>
```

Comment:
should be the same as in stars task
checklist

Reply:
fixed